File: LoanApp/ClientApp/src/app/models/loan.model.ts
import { ScheduleType } from "./schedule.model";

export class Loan {
    public Id: number = 0;
    public CustomerId: number = 0;
    public LoanNumber: string;
    public Amount: number = 0;
    public TransactionDate: string;
    public EstimatedDueDate: string;
    public Interest: number = 10;
    public AccountId: number = 0;
    public ScheduleType: ScheduleType
}

File: Models/LoanModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Models
{
    public class LoanModel
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public string LoanNumber { get; set; }
        public decimal Amount { get; set; }
        public DateTime TransactionDate  { get; set; }
        public DateTime EstimatedDueDate { get; set; }
        public decimal Interest { get; set; }
        public int AccountId { get; set; }
        public int ScheduleType  { get; set; }
    }
}
     